### ISSUE TITLE
Add code coverage badges

### ISSUE THREAD

**Issue by oneilsh** [monarch-initiative/monarchr#4]

**Comment by bschilder** [comment]
added here: #15

**Comment by oneilsh** [comment]
Thanks Brian - just making a note for this one that I have yet to link them to codecov.io

**Comment by oneilsh** [comment]
Hmm, so I could use some help getting the rworkflows checks and badges running properly. @bschilder, would you mind taking a look? I had previously removed Windows and OSX builds in rworkflows.yml as they were breaking (but not sure why), and I turned off run_rcmdcheck but I think it's ready to go now. I should be able to setup the link to codecov.io, but I don't remember how to trigger the codecov.yaml generation. And, I'm not sure if README.md is being properly built from README.Rmd.

Basically, I broke a lot of the rworkflows stuff to get the GH pages site to properly publish...

**Comment by bschilder** [comment]
The *codecov.yaml* should't be necessary anymore, and one rworkflows passes all the previous steps the codecov report should automatically be uploaded to Codecov and the badge will be updated.

That said, this only happens automatically when Codecov isn't havent problems, which seems to be happening more frequently lately. We just added a new feature to rworkflows that allows you to supply the CODECOV token (which can be stored in the monarchr repo as a GitHub secret), just in case we need it.

https://github.com/neurogenomics/rworkflows/pull/132

**Comment by bschilder** [comment]
The reason the Codecov badge isn't updating here could be due to a permissions issue with the monarchr repo, or the monarch-initiative org in general (I see none of the repos have been activated for codecov).
https://app.codecov.io/gh/monarch-initiative

Someone with admin permissions might need to follow some of these steps to let Codecov do its thing:
https://docs.codecov.com/docs/quick-start
